lib/v2/mclaren/radar.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    'mclaren.com': {
+        _name: 'Mclaren Racing',
+        www: [{
+            title: 'Mclaren Racing',
+            docs: 'https://docs.rsshub.app/new-media.html#_mclaren_racing',
+            source: '/racing/articles/'
```

Comment:
```suggestion
            source: '/racing/articles/',
```
That's why lint fails